### PR TITLE
[opt](brpc) Make the timeout of brpc checking configurable

### PR DESCRIPTION
**Pull request by mrhhsg**
### What problem does this PR solve?

Issue Number: close #xxx

Related PR: #xxx

Problem Summary:

### Release note

None

### Check List (For Author)

- Test 
    - [ ] Regression test
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [x] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [x] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [x] No.
    - [ ] Yes. 

- Does this need documentation?
    - [x] No.
    - [ ] Yes. 

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by mrhhsg** [comment]
run buildall

**Comment by mrhhsg** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]

**Review by yiguolei — DISMISSED** [review]

File: be/src/common/config.cpp [hunk 1]
```diff
@@ -554,6 +554,8 @@ DEFINE_Bool(enable_brpc_builtin_services, "true");
 // Enable brpc connection check
 DEFINE_Bool(enable_brpc_connection_check, "false");
 
+DEFINE_mInt64(brpc_connection_check_timeout_ms, "1000");
```
Comment (yiguolei) [inline be/src/common/config.cpp:557]: change default value to a larger timeout value, for example 10s

**Comment by mrhhsg** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Review by jacktengg — APPROVED** [review]

**Pull request merged**